Incorrect type in Zendesk Support (Ticket Metrics)

<!-- 
Welcome to Airbyte!
We're really appreciate your report and know that this will help us build an 
amazing tool. If you want to contribute yourself, you can find a good place 
to start by searching for the good-first-issues label or maybe... by trying 
to solve this one? (we can help debug this with you!)

Right now we are in alpha, so we're releasing versions a lot more frequently than 
normal. You can help us get to the root of the problem faster by filling out the
questionnaire below!

It's really important having all information and context. 
You can remove the examples bellow and fill out with your information.

--> 

## Enviroment
- **Airbyte version**:  0.30.20-alpha
- **OS Version / Instance**: Amazon Linux / EC2 t3a.large
- **Deployment**: example are Docker or Kubernetes deploy env
- **Source Connector and version**: Zendesk Support 0.1.3
- **Destination Connector and version**: Bigquery 0.4.1
- **Severity**: Medium
- **Step where error happened**: Sync job

## Current Behavior
dbt normalization failed when tried to transform initially_assigned_at as date

## Expected Behavior
initially_assigned_at supposed transformed to date time type

## Logs
*If applicable, please upload the logs from the failing operation. 
For sync jobs, you can download the full logs from the UI by going to the sync attempt page and 
clicking the download logs button at the top right of the logs display window.*

<details>
<summary>LOG</summary> 

```

2021-10-26 05:39:04 INFO () DefaultReplicationWorker(run):121 - Source thread complete.
2021-10-26 05:39:04 INFO () DefaultReplicationWorker(run):122 - Waiting for destination thread to join.
2021-10-26 05:39:04 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:04 [32mINFO[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):60 - {} - Airbyte message consumer: succeeded.
2021-10-26 05:39:04 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:04 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(close):143 - {} - Started closing all connections
2021-10-26 05:39:05 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:05 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):258 - {} - Waiting for jobs to be finished/closed
2021-10-26 05:39:14 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:14 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):275 - {} - Migration finished with no explicit errors. Copying data from tmp tables to permanent
2021-10-26 05:39:16 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:16 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(copyTable):339 - {} - successfully copied tmp table: GenericData{classInfo=[datasetId, projectId, tableId], {datasetId=zendesk_staging, tableId=_airbyte_tmp_ecv_ticket_metrics}} to final table: GenericData{classInfo=[datasetId, projectId, tableId], {datasetId=zendesk_staging, tableId=_airbyte_raw_ticket_metrics}}
2021-10-26 05:39:16 INFO () DefaultReplicationWorker(lambda$getDestinationOutputRunnable$3):231 - state in DefaultReplicationWorker from Destination: io.airbyte.protocol.models.AirbyteMessage@3f2965e1[type=STATE,log=<null>,spec=<null>,connectionStatus=<null>,catalog=<null>,record=<null>,state=io.airbyte.protocol.models.AirbyteStateMessage@31ee8c74[data={"ticket_metrics":{"updated_at":"2021-10-25T23:05:46Z"}},additionalProperties={}],additionalProperties={}]
2021-10-26 05:39:16 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:16 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):287 - {} - Removing tmp tables...
2021-10-26 05:39:17 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:17 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):289 - {} - Finishing destination process...completed
2021-10-26 05:39:17 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:17 [32mINFO[m i.a.i.b.IntegrationRunner(run):133 - {} - Completed integration: io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-10-26 05:39:17 INFO () DefaultAirbyteStreamFactory(lambda$create$0):53 - 2021-10-26 05:39:17 [32mINFO[m i.a.i.d.b.BigQueryDestination(main):338 - {} - completed destination: class io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-10-26 05:39:17 INFO () DefaultReplicationWorker(run):124 - Destination thread complete.
2021-10-26 05:39:17 INFO () DefaultReplicationWorker(run):152 - sync summary: io.airbyte.config.ReplicationAttemptSummary@101eff4e[status=completed,recordsSynced=205476,bytesSynced=188405795,startTime=1635203129351,endTime=1635226757305]
2021-10-26 05:39:17 INFO () DefaultReplicationWorker(run):159 - Source output at least one state message
2021-10-26 05:39:17 INFO () DefaultReplicationWorker(run):165 - State capture: Updated state to: Optional[io.airbyte.config.State@3c42ae7a[state={"ticket_metrics":{"updated_at":"2021-10-25T23:05:46Z"}}]]
2021-10-26 05:39:17 INFO () TemporalAttemptExecution(get):115 - Stopping cancellation check scheduling...
2021-10-26 05:39:17 INFO () SyncWorkflow$ReplicationActivityImpl(replicate):178 - sync summary: io.airbyte.config.StandardSyncOutput@f14ae5a[standardSyncSummary=io.airbyte.config.StandardSyncSummary@2156edb3[status=completed,recordsSynced=205476,bytesSynced=188405795,startTime=1635203129351,endTime=1635226757305],state=io.airbyte.config.State@3c42ae7a[state={"ticket_metrics":{"updated_at":"2021-10-25T23:05:46Z"}}],outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@cb616d0[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@211649f1[stream=io.airbyte.protocol.models.AirbyteStream@b5a55f7[name=ticket_metrics,jsonSchema={"type":["null","object"],"properties":{"id":{"type":["null","integer"]},"url":{"type":["null","string"]},"time":{"type":["null","string"]},"type":{"type":["null","string"]},"metric":{"type":["null","string"]},"status":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"reopens":{"type":["null","integer"]},"replies":{"type":["null","integer"]},"solved_at":{"type":["null","string"],"format":"date-time"},"ticket_id":{"type":["null","integer"]},"created_at":{"type":["null","string"],"format":"date-time"},"updated_at":{"type":["null","string"],"format":"date-time"},"assigned_at":{"type":["null","string"],"format":"date-time"},"instance_id":{"type":["null","integer"]},"group_stations":{"type":["null","integer"]},"assignee_stations":{"type":["null","integer"]},"status_updated_at":{"type":["null","string"],"format":"date-time"},"assignee_updated_at":{"type":["null","string"],"format":"date-time"},"requester_updated_at":{"type":["null","string"],"format":"date-time"},"initially_assigned_at":{"type":["null","string"],"format":"datetime"},"reply_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"latest_comment_added_at":{"type":["null","string"],"format":"date-time"},"on_hold_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"agent_wait_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"requester_wait_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"full_resolution_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}},"first_resolution_time_in_minutes":{"type":["null","object"],"properties":{"business":{"type":["null","integer"]},"calendar":{"type":["null","integer"]}}}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=true,defaultCursorField=[updated_at],sourceDefinedPrimaryKey=[[id]],namespace=<null>,additionalProperties={}],syncMode=incremental,cursorField=[updated_at],destinationSyncMode=append_dedup,primaryKey=[[id]],additionalProperties={}]],additionalProperties={}]]
2021-10-26 05:39:17 INFO () TemporalAttemptExecution(get):94 - Executing worker wrapper. Airbyte version: 0.30.20-alpha
2021-10-26 05:39:17 WARN () Databases(createPostgresDatabaseWithRetry):38 - Waiting for database to become available...
2021-10-26 05:39:17 INFO () JobsDatabaseInstance(lambda$static$2):25 - Testing if jobs database is ready...
2021-10-26 05:39:17 INFO () Databases(createPostgresDatabaseWithRetry):55 - Database available!
2021-10-26 05:39:17 INFO () DefaultNormalizationWorker(run):46 - Running normalization.
2021-10-26 05:39:17 INFO () DefaultNormalizationRunner(runProcess):107 - Running with normalization version: airbyte/normalization:0.1.52
2021-10-26 05:39:17 INFO () LineGobbler(voidCall):65 - Checking if airbyte/normalization:0.1.52 exists...
2021-10-26 05:39:17 INFO () LineGobbler(voidCall):65 - airbyte/normalization:0.1.52 was found locally.
2021-10-26 05:39:17 INFO () DockerProcessFactory(create):127 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/22932/2/normalize --network host --log-driver none airbyte/normalization:0.1.52 run --integration-type bigquery --config destination_config.json --catalog destination_catalog.json
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - Running: transform-config --config destination_config.json --integration-type bigquery --out /data/22932/2/normalize
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - Namespace(config='destination_config.json', integration_type=<DestinationType.bigquery: 'bigquery'>, out='/data/22932/2/normalize')
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - transform_bigquery
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - Running: transform-catalog --integration-type bigquery --profile-config-dir /data/22932/2/normalize --catalog destination_catalog.json --out /data/22932/2/normalize/models/generated/ --json-column _airbyte_data
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - Processing destination_catalog.json...
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_ab1.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_ab2.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_ab3.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_ab4.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_scd.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics.sql from ticket_metrics
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_status_ab1.sql from ticket_metrics/status
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_status_ab2.sql from ticket_metrics/status
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_status_ab3.sql from ticket_metrics/status
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_status.sql from ticket_metrics/status
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_reply_time_in_minutes_ab1.sql from ticket_metrics/reply_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_reply_time_in_minutes_ab2.sql from ticket_metrics/reply_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_reply_time_in_minutes_ab3.sql from ticket_metrics/reply_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_reply_time_in_minutes.sql from ticket_metrics/reply_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_on_hold_time_in_minutes_ab1.sql from ticket_metrics/on_hold_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_on_hold_time_in_minutes_ab2.sql from ticket_metrics/on_hold_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_on_hold_time_in_minutes_ab3.sql from ticket_metrics/on_hold_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_on_hold_time_in_minutes.sql from ticket_metrics/on_hold_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_agent_wait_time_in_minutes_ab1.sql from ticket_metrics/agent_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_agent_wait_time_in_minutes_ab2.sql from ticket_metrics/agent_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_agent_wait_time_in_minutes_ab3.sql from ticket_metrics/agent_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_agent_wait_time_in_minutes.sql from ticket_metrics/agent_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_requester_wait_time_in_minutes_ab1.sql from ticket_metrics/requester_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_requester_wait_time_in_minutes_ab2.sql from ticket_metrics/requester_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_requester_wait_time_in_minutes_ab3.sql from ticket_metrics/requester_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_requester_wait_time_in_minutes.sql from ticket_metrics/requester_wait_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_full_resolution_time_in_minutes_ab1.sql from ticket_metrics/full_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_full_resolution_time_in_minutes_ab2.sql from ticket_metrics/full_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_full_resolution_time_in_minutes_ab3.sql from ticket_metrics/full_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_full_resolution_time_in_minutes.sql from ticket_metrics/full_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_first_resolution_time_in_minutes_ab1.sql from ticket_metrics/first_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_first_resolution_time_in_minutes_ab2.sql from ticket_metrics/first_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_ctes/zendesk_staging/ticket_metrics_first_resolution_time_in_minutes_ab3.sql from ticket_metrics/first_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 -   Generating airbyte_tables/zendesk_staging/ticket_metrics_first_resolution_time_in_minutes.sql from ticket_metrics/first_resolution_time_in_minutes
2021-10-26 05:39:18 INFO () LineGobbler(voidCall):65 - detected no config file for ssh, assuming ssh is off.
2021-10-26 05:39:19 INFO () LineGobbler(voidCall):65 - Running with dbt=0.19.0
2021-10-26 05:39:22 INFO () LineGobbler(voidCall):65 - [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
2021-10-26 05:39:22 INFO () LineGobbler(voidCall):65 - There are 1 unused configuration paths:
2021-10-26 05:39:22 INFO () LineGobbler(voidCall):65 - - models.airbyte_utils.generated.airbyte_views
2021-10-26 05:39:22 INFO () LineGobbler(voidCall):65 - 
2021-10-26 05:39:23 INFO () LineGobbler(voidCall):65 - Found 34 models, 0 tests, 0 snapshots, 0 analyses, 440 macros, 0 operations, 0 seed files, 1 source, 0 exposures
2021-10-26 05:39:23 INFO () LineGobbler(voidCall):65 - 
2021-10-26 05:39:23 INFO () LineGobbler(voidCall):65 - 05:39:23 | Concurrency: 32 threads (target='prod')
2021-10-26 05:39:23 INFO () LineGobbler(voidCall):65 - 05:39:23 | 
2021-10-26 05:39:24 INFO () LineGobbler(voidCall):65 - 05:39:24 | 1 of 9 START table model zendesk_staging.ticket_metrics_scd.................................................. [RUN]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 1 of 9 ERROR creating table model zendesk_staging.ticket_metrics_scd......................................... [[31mERROR[0m in 2.52s]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 2 of 9 SKIP relation zendesk_staging.ticket_metrics.......................................................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 3 of 9 SKIP relation zendesk_staging.ticket_metrics_agent_wait_time_in_minutes............................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 5 of 9 SKIP relation zendesk_staging.ticket_metrics_on_hold_time_in_minutes.................................. [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 7 of 9 SKIP relation zendesk_staging.ticket_metrics_full_resolution_time_in_minutes.......................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 8 of 9 SKIP relation zendesk_staging.ticket_metrics_status................................................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 4 of 9 SKIP relation zendesk_staging.ticket_metrics_first_resolution_time_in_minutes......................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 6 of 9 SKIP relation zendesk_staging.ticket_metrics_requester_wait_time_in_minutes........................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 9 of 9 SKIP relation zendesk_staging.ticket_metrics_reply_time_in_minutes.................................... [[33mSKIP[0m]
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | 
2021-10-26 05:39:26 INFO () LineGobbler(voidCall):65 - 05:39:26 | Finished running 9 table models in 3.42s.
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - 
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - [31mCompleted with 1 error and 0 warnings:[0m
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - 
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - [33mDatabase Error in model ticket_metrics_scd (models/generated/airbyte_tables/zendesk_staging/ticket_metrics_scd.sql)[0m
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 -   Invalid date: '2019-01-01T01:06:27.000000Z'
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 -   compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/zendesk_staging/ticket_metrics_scd.sql
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - 
2021-10-26 05:39:27 INFO () LineGobbler(voidCall):65 - Done. PASS=0 WARN=0 ERROR=1 SKIP=8 TOTAL=9
2021-10-26 05:39:27 INFO () TemporalAttemptExecution(lambda$getWorkerThread$1):148 - Completing future exceptionally...
io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:60) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:145) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:57) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	... 3 more
	Suppressed: io.airbyte.workers.WorkerException: Normalization process wasn't successful
		at io.airbyte.workers.normalization.DefaultNormalizationRunner.close(DefaultNormalizationRunner.java:135) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:45) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:145) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at java.lang.Thread.run(Thread.java:832) [?:?]
2021-10-26 05:39:27 INFO () TemporalAttemptExecution(get):115 - Stopping cancellation check scheduling...
2021-10-26 05:39:27 WARN () POJOActivityTaskHandler$POJOActivityImplementation(execute):243 - Activity failure. ActivityId=b574f0de-68d6-3994-ab46-35a60a14b58c, activityType=Normalize, attempt=1
java.util.concurrent.ExecutionException: io.airbyte.workers.WorkerException: Normalization Failed.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2063) ~[?:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:113) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.temporal.SyncWorkflow$NormalizationActivityImpl.normalize(SyncWorkflow.java:299) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at jdk.internal.reflect.GeneratedMethodAccessor169.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler$POJOActivityInboundCallsInterceptor.execute(POJOActivityTaskHandler.java:277) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler$POJOActivityImplementation.execute(POJOActivityTaskHandler.java:216) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler.handle(POJOActivityTaskHandler.java:181) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:192) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:154) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:73) ~[temporal-sdk-1.0.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:60) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:145) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	... 1 more
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:57) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:145) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
	... 1 more
	Suppressed: io.airbyte.workers.WorkerException: Normalization process wasn't successful
		at io.airbyte.workers.normalization.DefaultNormalizationRunner.close(DefaultNormalizationRunner.java:135) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:45) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:145) ~[io.airbyte-airbyte-workers-0.30.20-alpha.jar:?]
		at java.lang.Thread.run(Thread.java:832) [?:?]

```

</details>

## Steps to Reproduce
1. set catalog schema to `ticket metrics`

## Are you willing to submit a PR?
Yes

--- a/airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/79c1aa37-dae3-42ae-b333-d1c105477715.json
+++ b/airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/79c1aa37-dae3-42ae-b333-d1c105477715.json
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "79c1aa37-dae3-42ae-b333-d1c105477715",
   "name": "Zendesk Support",
   "dockerRepository": "airbyte/source-zendesk-support",
-  "dockerImageTag": "0.1.3",
+  "dockerImageTag": "0.1.4",
   "documentationUrl": "https://docs.airbyte.io/integrations/sources/zendesk-support",
   "icon": "zendesk.svg"
 }

--- a/airbyte-config/init/src/main/resources/seed/source_definitions.yaml
+++ b/airbyte-config/init/src/main/resources/seed/source_definitions.yaml
@@ -572,7 +572,7 @@
 - name: Zendesk Support
   sourceDefinitionId: 79c1aa37-dae3-42ae-b333-d1c105477715
   dockerRepository: airbyte/source-zendesk-support
-  dockerImageTag: 0.1.3
+  dockerImageTag: 0.1.4
   documentationUrl: https://docs.airbyte.io/integrations/sources/zendesk-support
   icon: zendesk.svg
   sourceType: api

--- a/airbyte-integrations/connectors/source-zendesk-support/Dockerfile
+++ b/airbyte-integrations/connectors/source-zendesk-support/Dockerfile
@@ -25,5 +25,5 @@ COPY source_zendesk_support ./source_zendesk_support
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.4
 LABEL io.airbyte.name=airbyte/source-zendesk-support

--- a/airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_metrics.json
+++ b/airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_metrics.json
@@ -132,7 +132,7 @@
     },
     "initially_assigned_at": {
       "type": ["null", "string"],
-      "format": "datetime"
+      "format": "date-time"
     },
     "assigned_at": {
       "type": ["null", "string"],

--- a/docs/integrations/sources/zendesk-support.md
+++ b/docs/integrations/sources/zendesk-support.md
@@ -97,6 +97,7 @@ We recommend creating a restricted, read-only key specifically for Airbyte acces
 
 | Version | Date | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| `0.1.4` | 2021-10-26 | [7377](https://github.com/airbytehq/airbyte/pull/7377) | fix initially_assigned_at type in ticket metrics |
 | `0.1.3` | 2021-10-17 | [7097](https://github.com/airbytehq/airbyte/pull/7097) | correction of spec file |
 | `0.1.2` | 2021-10-16 | [6513](https://github.com/airbytehq/airbyte/pull/6513) | fixed comments stream |
 | `0.1.1` | 2021-09-02 | [5787](https://github.com/airbytehq/airbyte/pull/5787) | fixed incremental logic for the ticket_comments stream |

